src/NServiceBus.RabbitMQ/Receiving/ModelExtensions.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace NServiceBus.Transport.RabbitMQ
+{
+    using System.Threading.Tasks;
+    using global::RabbitMQ.Client;
+
+    static class ModelExtensions
+    {
+        class MessageState
+        {
+            public IModel Channel { get; set; }
+
+            public ulong DeliveryTag { get; set; }
+        }
+
+        public static Task BasicAck(this IModel channel, ulong deliveryTag, bool multiple, TaskScheduler scheduler) =>
+            TaskEx.StartNew(
+                new MessageState { Channel = channel, DeliveryTag = deliveryTag },
+                state =>
+                {
+                    var messageState = (MessageState)state;
+                    messageState.Channel.BasicAck(messageState.DeliveryTag, multiple);
```

Comment:
Using `multiple` directly here means you've re-introduced a closure. We need to add an additional property to `MessageState` to pass it in, too.

Or, we could drop the `multiple` parameter and hard-code the `false` again.